person.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def of_age?
+    @age >= 18
+  end
+
```

Comment:
- [ ] This method is private which is a requirement so please make it private :blush:

Hint: You can make it private by adding the following line at line 14 :+1:
**private of_age?**